fala/templates/adviser/results.html

```diff
@@ -11,6 +11,21 @@
   {% if FEATURE_FLAG_SURVEY_MONKEY %}
     {% include 'adviser/_research_banner.html' %}
   {% endif %}
+  {% if tailored_results %}
+    <form action="/single-category-search" method="GET" novalidate>
+      <input type="hidden" name="postcode" value="{{ form.postcode.value() }}">
+      <input type="hidden" name="name" value="{{ form.name.value() }}">
+      {% for value, label_text in form.categories.field.choices %}
+        {% if label_text in form|category_selection_list %}
+          <input type="hidden" name="categories" value="{{ value }}">
+        {% endif %}
+      {% endfor %}
+      <a href="/single-category-search?postcode={{ form.postcode.value() }}&name={{ form.name.value() }}{% for value, label_text in form.categories.field.choices %}{% if label_text in form|category_selection_list %}&categories={{ value }}{% endif %}{% endfor %}" class="govuk-back-link govuk-!-margin-bottom-2 govuk-!-display-none-print">
+        Back
+      </a>
+    </form>
+  {% endif %}
+
   <div class="govuk-grid-column-full">
```

Comment:
I think this needs to be above `FEATURE_FLAG_SURVEY_MONKEY`, wrapped into a `div` and more margin added to the bottom. Update code below:

```
  {% if tailored_results %}
    <div class="govuk-grid-column-full">
      <form action="/single-category-search" method="GET" novalidate>
        <input type="hidden" name="postcode" value="{{ form.postcode.value() }}">
        <input type="hidden" name="name" value="{{ form.name.value() }}">
        {% for value, label_text in form.categories.field.choices %}
          {% if label_text in form|category_selection_list %}
            <input type="hidden" name="categories" value="{{ value }}">
          {% endif %}
        {% endfor %}
        <a href="/single-category-search?postcode={{ form.postcode.value() }}&name={{ form.name.value() }}{% for value, label_text in form.categories.field.choices %}{% if label_text in form|category_selection_list %}&categories={{ value }}{% endif %}{% endfor %}" class="govuk-back-link govuk-!-margin-bottom-5 govuk-!-display-none-print">
          Back
        </a>
      </form>
    </div>
  {% endif %}
  {% if FEATURE_FLAG_SURVEY_MONKEY %}
    {% include 'adviser/_research_banner.html' %}
  {% endif %}
```
It currently looks like this. 
![Screenshot 2024-12-12 at 16 39 16](https://github.com/user-attachments/assets/2ff1b5ef-ccb1-4633-b270-3b8d61785cc7)

Should we as a test for it?

Reply:
i have amended the code. I have the flag turned off so didn't catch this so I appreciate the attention to detail.